lib/pages/settings/view/feedback_form.dart

```diff
@@ -0,0 +1,166 @@
+          ],
+          body: ListView(
+            padding: const EdgeInsets.all(16),
+            children: <Widget>[
+              Padding(
+                padding: const EdgeInsets.only(top: 12, bottom: 12),
+                child: Container(
+                    height: MediaQuery.of(context).size.height / 4,
+                    child: Image.asset(
+                        'assets/illustrations/undraw_feedbackform.png')),
+              ),
+              Row(
+                children: [
+                  Expanded(
+                    child: Row(
+                      children: [
+                        const SizedBox(width: 12),
+                        Icon(FeatherIcons.filter,
+                            color: Theme.of(context).formIconColor),
+                        const SizedBox(width: 12),
+                        Text(
+                          S.current.labelReportType,
+                          style: Theme.of(context)
+                              .textTheme
+                              .subtitle1
+                              .copyWith(fontWeight: FontWeight.w400),
+                        ),
+                      ],
+                    ),
+                  ),
+                  ChoiceChip(
+                    key: const ValueKey('choice_feedback'),
+                    label: Text(
+                      S.current.labelFeedback,
+                      style: Theme.of(context).chipTextStyle(selected: true),
+                    ),
+                    selected: _feedbackSelected,
+                    onSelected: (selected) => setState(() {
+                      _feedbackSelected = true;
+                      if (selected) {
+                        _issueSelected = false;
+                      }
+                    }),
+                  ),
+                  const SizedBox(width: 10),
+                  ChoiceChip(
+                    key: const ValueKey('choice_issue'),
+                    label: Text(
+                      S.current.labelIssue,
+                      style: Theme.of(context).chipTextStyle(selected: true),
+                    ),
+                    selected: _issueSelected,
+                    onSelected: (selected) => setState(() {
+                      _issueSelected = true;
+                      if (selected) {
+                        _feedbackSelected = false;
+                      }
+                    }),
+                  ),
+                  const SizedBox(width: 10),
+                ],
+              ),
+              TextFormField(
+                key: const ValueKey('issue'),
+                controller: issueController,
+                validator: (value) {
+                  if (value == null || value.isEmpty) {
+                    return S.current.messageEnterText;
+                  }
+                  return null;
+                },
+                keyboardType: TextInputType.multiline,
+                minLines: 1,
+                maxLines: 10,
```

Comment:
Do we need a limit here? 10 lines doesn't seem like a lot if they're e.g. describing how to reproduce an issue.

Reply:
The limit is just for how much the widget for the text field can expand down. If maxLines is removed, the text field won't expand anymore and the text will just get hidden and become hard to edit. The user is not restricted to write 10 lines of text.

Reply:
Gotcha, makes sense.